### PR TITLE
statistic: total products, total product orderd, total orderd, total re…

### PR DESCRIPTION
**Pull request by at-thaitran**
…venue

## Description
stistic: total products, total product orderd, total orderd, total revenue with orders.status = 2(accpeted)
## Reference Requirements
- Task ID: TI-1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)
```
mysql> EXPLAIN select count(*) as aggregate from `products` where `products`.`deleted_at` is null \G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: products
   partitions: NULL
         type: ALL
possible_keys: NULL
          key: NULL
      key_len: NULL
          ref: NULL
         rows: 20
     filtered: 10.00
        Extra: Using where
1 row in set, 1 warning (0.00 sec)

ERROR: 
No query specified
```
```
mysql> EXPLAIN select count(*) as aggregate from `orders` where `status` = '2' \G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: orders
   partitions: NULL
         type: ALL
possible_keys: NULL
          key: NULL
      key_len: NULL
          ref: NULL
         rows: 35
     filtered: 10.00
        Extra: Using where
1 row in set, 1 warning (0.00 sec)

ERROR: 
No query specified
```
```
mysql> EXPLAIN select sum(`total`) as aggregate from `orders` where `status` = '2' \G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: orders
   partitions: NULL
         type: ALL
possible_keys: NULL
          key: NULL
      key_len: NULL
          ref: NULL
         rows: 35
     filtered: 10.00
        Extra: Using where
1 row in set, 1 warning (0.00 sec)

ERROR: 
No query specified
```
```
mysql> EXPLAIN select sum(`quantity`) as aggregate from `order_details` where exists (select * from `orders` where `order_details`.`order_id` = `orders`.`id` and `status` = '2') \G;
*************************** 1. row ***************************
           id: 1
  select_type: PRIMARY
        table: order_details
   partitions: NULL
         type: ALL
possible_keys: NULL
          key: NULL
      key_len: NULL
          ref: NULL
         rows: 20
     filtered: 100.00
        Extra: Using where
*************************** 2. row ***************************
           id: 2
  select_type: DEPENDENT SUBQUERY
        table: orders
   partitions: NULL
         type: eq_ref
possible_keys: PRIMARY
          key: PRIMARY
      key_len: 4
          ref: food_drink.order_details.order_id
         rows: 1
     filtered: 10.00
        Extra: Using where
2 rows in set, 2 warnings (0.00 sec)

ERROR: 
No query specified

```

### CODE REVIEW

**Comment by at-dungle** [comment]
LGTM

File: app/Http/Controllers/Admin/HomeController.php [hunk 1]
```diff
@@ -14,6 +17,12 @@ class HomeController extends Controller
     */
     public function index()
     {
-        return view('admin.home.index');
+        $totalProduct = Product::count();
+        $totalOrder = Order::where('status', 2)->count();
+        $totalRevenue = Order::where('status', 2)->sum('total');
+        $totalProductOrderd = OrderDetail::whereHas('order', function ($query) {
+            $query->where('status', '=', 2);
```
Comment (at-thinhnguyen) [inline app/Http/Controllers/Admin/HomeController.php]: số 2 define trong model nhá

File: app/Http/Controllers/Admin/HomeController.php [hunk 2]
```diff
@@ -14,6 +17,12 @@ class HomeController extends Controller
     */
     public function index()
     {
-        return view('admin.home.index');
+        $totalProduct = Product::count();
+        $totalOrder = Order::where('status', Order::ACCPECTED)->count();
+        $totalRevenue = Order::where('status', Order::ACCPECTED)->sum('total');
+        $totalProductOrderd = OrderDetail::whereHas('order', function ($query) {
+            $query->where('status', '=', Order::ACCPECTED);
```
Comment (at-dungle) [inline app/Http/Controllers/Admin/HomeController.php]: ACCPECTED ?

**Comment by at-dungle** [comment]
LGTM

**Pull request merged**